src/java.base/share/classes/java/security/spec/PSSParameterSpec.java

```diff
@@ -95,15 +95,15 @@
     public static final int TRAILER_FIELD_BC = 1;
 
     /**
-     * The PSS parameter set with all default values
+     * The PSS parameter set with all default values.
      * @deprecated This field uses the default values defined in the PKCS #1
      *         standard. Some of these defaults are no longer recommended due
      *         to advances in cryptanalysis -- see the
      *         <a href="https://tools.ietf.org/rfc/rfc8017.txt">PKCS#1 v2.2</a>
      *         standard for more details. Thus, it is recommended to create
      *         a new {@code PSSParameterSpec} with the desired parameter values
      *         using
```

Comment:
s/using/using the/

Reply:
Yes, thanks. Need to re-read the whole sentence again even with minor changes...